### PR TITLE
Fix for  python package dependencies, concurrent operations , container build issues

### PR DESCRIPTION
**Pull request by wdurairaj**
Change in requirement.txt to address issue #27
https://github.com/hpe-storage/python-hpedockerplugin/issues/27

### CODE REVIEW

**Comment by garthmbooth** [comment]
William, the changes look good to me. Once you've done appropriate regression tests, I'm ok to merge.

**Comment by garthmbooth** [comment]
William, can you make sure the changes in the commit [use inline lock/unlock in place of functions...](https://github.com/hpe-storage/python-hpedockerplugin/pull/28/commits/ba07eb94cc563239f860057e72edee0501cf3186) is included in this PR? It appears it is not.

**Comment by wdurairaj** [comment]
@garthmbooth  
Hi Garth,
I have reintroduced the locking fix which was original missed by the lockingUnitTest-Fixes branch and updated this pull request. Please review. Meanwhile I will be testing this changes

Thanks.
William

File: hpedockerplugin/hpe_storage_api.py [hunk 1]
```diff
@@ -1,561 +1,658 @@
...
+        # hosts at the same time.
+        self._etcd.update_vol(volid, 'path_info', None)
+
+        LOG.info(_LI('path for volume: %(name)s, was successfully removed: '
+                     '%(path_name)s'), {'name': volname,
+                                        'path_name': path_name})
+
+        response = json.dumps({u"Err": ''})
+        return response
+
```
Comment (garthmbooth) [inline hpedockerplugin/hpe_storage_api.py]: We need to remove or refactor the _lock_volume and _unlock_volume functions as they don't work as expected. They are currently not used by the plugin.

**Comment by garthmbooth** [comment]
William,

Overall, the changes to add back in the inline lock/unlock changes look good.

Garth

File: hpedockerplugin/hpe_storage_api.py (hunk 1 above)
Comment (wdurairaj) [inline hpedockerplugin/hpe_storage_api.py]: I have removed the unused function definitions.

**Pull request merged**